### DISCUSSION TITLE
Never got an answer on my first attempt, lets hope this one has more luck - Why is my displayed consumption so low?

### DISCUSSION THREAD

**Discussion by SylvainGa** [teslamate-org/teslamate#2094] [General]
![image](https://user-images.githubusercontent.com/49377866/138618465-4f2dcecf-6962-4ed7-a27e-17f44deda6d9.png)

This is way lower than what is displayed in car. How can I fix it?

Thanks.

**Comment by SylvainGa** [comment]
I don't know why, but 'efficiency' under 'cars' was very low (0.04032) so "range_diff * car_efficiency" was skewed.

My last 20 km drive according the the display in the car (last drive) showed 157 Wh/km so by multiplying both, I got 3,140 kWh for that drive. 

The 'drives' table row for that drive had 293.32 km and 254.71 start_rated_range_km and end_rated_range_km respectively, so a range_diff of 38.61 km. This gave me an efficiency of 0.081326, which is what I've set the car's efficiency field with and then consumption was more realistic

![image](https://user-images.githubusercontent.com/49377866/139001899-5a818dc1-cd16-44cf-a5b4-d309ce1e4980.png)

However :-( the car's efficiency went back to 0.04032. I know from the FAQ that it's calculated and gets better with time but I have 207 drives recorded and yet, it's way off. Anybody know how I can fix this, permanently?
 
Thanks

PS. I converted from Q/A to General since I never got an answer to my question so there was no point to leave it there. My next step is to make it an issue if I can't get it resolved in the discussions area.

**Comment by OzBrown** [comment]
When you look at each of the drives what do they show for Consumption against Klm driven, as that may help in seeing what the data is showing.
